Add mission to take refugees to Humanika (#9159)
endless-sky/endless-sky: data/human/human missions.txt

				clear "Blind Man from Martini: label looking"
			`	With that, he grabs his cane and his luggage, and follows a hospital attendant toward his daughter's room, leaving you in the reception.`

mission "FW Refugees to Humanika"
	name "Transport Refugee Family"
	description "Transport a family of <bunks> refugees and <cargo> to <destination>. Payment is <payment>."
	minor
	source

> 		attributes "dirt belt" rim south

Reviewer: ```suggestion
		attributes "dirt belt" "rim" "south"
```

		not government Quarg
		not near Tarazed 1
	destination Humanika
	to offer
		has "event: war begins"
		not "chosen sides"
		random < 15
	to fail

> 		has "event: joined the free worlds"

Reviewer: ```suggestion
		has "chosen sides"
```

Author: Doesn't `"chosen sides"` happen before the passengers would find out that the captain chose sides? So why would they abandon the captain then?

	passengers 3
	cargo "household goods" 2
	on offer
		conversation
			`A young family of three is walking around the spaceport talking to merchant captains. From the looks of it, they're not having much luck.`
			choice
				`	(Approach them.)`
				`	(Ignore them.)`
					defer

> 			`	The parents introduce themselves as Penny Little and Rosa Perkins. Their young child is named Eduardo. "Any chance you're willing to take us and <tons> of our belongings to the Quarg planet <destination>? We're concerned that the war between the Republic and the Free Worlds is going to get dangerous, and we want to raise Eduardo somewhere safe. We can pay <payment>."`

Author: I don't really see how not specifying the speaker is confusing, but I'm not going to keep pushing the point. I applied all these changes, but I moved "Penny asks" to just before the quote. The way it was in the middle looked like it was its own sentence to me, which I found confusing.

			label "main choice"
			choice
				`	"Why were all those other captains refusing to take you?"`
					to display
						not "FW Refugees to Humanika: other captains"
					goto "other captains"

> 				`	"What do you mean, 'Quarg planet'?"`

Reviewer 2: ```suggestion
				`	"What do you mean, 'Quarg planet?'"`
```

Author: See https://github.com/endless-sky/endless-sky/pull/9159#discussion_r1299249287 above, I'm pretty sure this is correct.

Reviewer 2: This is actually an American vs British English thing. The game is written in American English; as such, the `?` should be inside both quotation marks.

Author: That's for commas and periods, not question marks, I think. I think question marks are the same in both styles. E.g., https://www.thepunctuationguide.com/british-versus-american-style.html says:

> The above examples also show that the American style places commas and periods inside the quotation marks, even if they are not in the original material. British style (more sensibly) places unquoted periods and commas outside the quotation marks. For all other punctuation, the British and American styles are in agreement: unless the punctuation is part of the quoted material, it goes outside the quotation marks.

P.S. I'm American btw, though I read enough English from other places that I don't always write in American style without consciously thinking about it.

					to display
						not "FW Refugees to Humanika: quarg"
						not "First Contact: Quarg: offered"
					goto quarg
				`	"The Quarg let humans live on one of their planets?"`
					to display
						not "FW Refugees to Humanika: quarg"
						has "First Contact: Quarg: offered"
						not "visited planet: Humanika"
					goto quarg
				`	"Sure, I can take you."`
					accept
				`	"Sorry, I'm not headed that way."`
					decline
			label "other captains"
			action
				set "FW Refugees to Humanika: other captains"

> 			`	"I think many of them are a bit distrustful of the Quarg, and people are just generally more on edge these days with the tensions between the Free Worlds and the Republic, so they're less interested in flying somewhere unfamiliar. While I do understand those feelings, it seems to me that humans have done much worse to each other than the Quarg have ever done to us."`

Reviewer: ```suggestion
			`	"I think many of them are a bit distrustful of the Quarg. Plus, people are just generally more on edge these days with the tensions between the Free Worlds and the Republic, so they're less interested in flying somewhere unfamiliar. While I do understand those feelings, it seems to me that humans have done much worse to each other than the Quarg have ever done to us."`
```

Author: Done.

				goto "main choice"
			label "quarg"
			action
				set "FW Refugees to Humanika: quarg"

> 			`	She looks a bit surprised that you don't already know. "The Quarg are an advanced and apparently peaceful alien species that live in a corner of human space, near Tarazed. I honestly don't know that much else about them, but I've heard they're happy to talk to anybody who visits them. You could try that if you want to know more."`

Reviewer: ```suggestion
			`	She looks a bit surprised that you don't already know. "The Quarg are an advanced alien species that live in a corner of human space, near Tarazed. I honestly don't know that much else about them, but I've heard that they prefer to stay out of human matters. You could try talking to them if you want to know more."`
```

Author: Done. From a gameplay perspective, do you think "talking to them" is clear enough that the player should land on one of their moons/stations, as opposed to hailing one of their ships?

Reviewer: It's clear enough.

				to display
					not "First Contact: Quarg: offered"
			`	"There's a planet they control that has too much gravity for them, but is just right for humans, so they let humans settle there under their protection. Or at least that's what I've heard."`
				goto "main choice"
	on visit
		dialog phrase "generic cargo and passenger on visit"
	on complete
		payment
		payment 20000
		dialog "Penny and Rosa thank you for getting them safely to <destination>, handing you <payment> for your troubles. You wish them and Eduardo the best of luck starting their new life away from a potential war in human space."

> 		log "Minor People" "Penny Little, Rosa Perkins, and Eduardo" `A young family who wanted to escape a potential war in human space. Took them to the Quarg planet Humanika, where the Quarg let humans settle.`

Reviewer 2: ```suggestion
		log "Minor People" "Penny Little, Rosa Perkins, and Eduardo" `A young family who wanted to escape a potential war in human space. Took them to the planet Humanika, where the Quarg let humans settle.`
```

Author: Good idea, the end of the sentence already makes it clear that it's a Quarg planet, so no reason to state it twice. Done.
